data/json/dreams.json

```diff
@@ -224,6 +233,7 @@
     "type": "dream",
     "messages": [
       "You have a confusing dream of growing in a garden.",
+      "You remember a strange song lyric you once heard.  \"The ardor of arboreality is an adventure we have spurned\" - it is clearly time to overturn a new leaf.",
```

Comment:
This might be overcomplicated. Most of the mutant dreams are about being the thing you're turning into, so I think plant dreams for exemple could be about craving sunlight, feeling the wind in your leafs, fearing the bugs eating you, spreading your roots in the soill etc... but not really about cryptic messages from song lyrics.

Reply:
makes sense